install/migrations/update_10.0.x_to_11.0.0/profiles.php

```diff
@@ -47,3 +49,14 @@
     ]
 );
 $migration->addKey('glpi_profiles', 'last_rights_update');
+
+$migration->addField(
+    'glpi_profiles',
+    'use_mentions',
+    'int',
+    [
+        'null' => false,
+        'value' => UserMention::USER_MENTION_FULL,
```

Comment:
Please use only hardcoded values in migrations. The constant is unlikely to change, but if it does, then it would change the migration behaviour and can have unexpected side effects.

```suggestion
        'value' => '1',
```